README.md

```diff
@@ -1,56 +1,19 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Set up a "Hello Microverse" project
 
+> In this project, I set up a "Hello world" repository with no complex coding included. My goal is to master all of the tools and best practices I I have learned about regarding Git, Github Flow, Linters, Professional Repo Documentation and Code Review. I will be using this skills learnt in all Microverse projects and most likely in my future job as well!
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
+- Html
+- CSS
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@XsidX](https://github.com/XsidX)
+- Twitter: [@twitterhandle](https://twitter.com/_sid_o)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sidney-kaguli-0116801a6/)
```

Comment:
- [ ] OPTIONAL: I would suggest you change your link names instead of keeping the initial ones for a better README design. Check the model below:
```suggestion
- Twitter: [@_sid_o](https://twitter.com/_sid_o)
- LinkedIn: [Sidney-Kaguli](https://www.linkedin.com/in/sidney-kaguli-0116801a6/)
```

Reply:
That's right, I forgot to customize those two link names.